apps/files/src/components/FilesAppBar.vue

```diff
@@ -349,17 +353,25 @@ export default {
         p.then(() => {
           this.createFile = false
           this.$_ocFilesFolder_getFolder()
+          this.fileFolderCreationLoading = false
+          if (this.newFileAction) {
+            // not cool - needs refactoring
+            this.$nextTick(() => {
+              this.openFile({
```

Comment:
I'm confused about what the difference is between `openFile` and `openFileAction` and also why we call both after each other in a synchronous manner.

Please rename the functions to more explicit names or add comments here.

Reply:
Part of the tech dept - refs TODO above ... will open ticket

Reply:
ref https://github.com/owncloud/phoenix/issues/2300

Reply:
https://github.com/owncloud/phoenix/issues/2300